webapp/channels/src/components/channel_layout/center_channel/center_channel.tsx

```diff
@@ -2,43 +2,35 @@
 // See LICENSE.txt for license information.
 
 import classNames from 'classnames';
-import React from 'react';
+import React, {lazy} from 'react';
 import {Route, Switch, Redirect} from 'react-router-dom';
 
 import {makeAsyncComponent} from 'components/async_load';
-import ChannelIdentifierRouter from 'components/channel_layout/channel_identifier_router';
-import PlaybookRunner from 'components/channel_layout/playbook_runner';
 import LoadingScreen from 'components/loading_screen';
-import PermalinkView from 'components/permalink_view';
 
 import {IDENTIFIER_PATH_PATTERN, ID_PATH_PATTERN, TEAM_NAME_PATH_PATTERN} from 'utils/path';
 
 import type {OwnProps, PropsFromRedux} from './index';
 
-const LazyChannelHeaderMobile = makeAsyncComponent(
-    'LazyChannelHeaderMobile',
-    React.lazy(() => import('components/channel_header_mobile')),
-);
-
-const LazyGlobalThreads = makeAsyncComponent(
-    'LazyGlobalThreads',
-    React.lazy(() => import('components/threading/global_threads')),
+const ChannelHeaderMobile = makeAsyncComponent('ChannelHeaderMobile', lazy(() => import('components/channel_header_mobile')));
+const SidebarMobileRightMenu = makeAsyncComponent('SidebarMobileRightMenu', lazy(() => import('components/sidebar_mobile_right_menu')));
+const GlobalThreads = makeAsyncComponent('GlobalThreads', lazy(() => import('components/threading/global_threads')),
     (
         <div className='app__content'>
             <LoadingScreen/>
         </div>
     ),
 );
-
-const LazyDrafts = makeAsyncComponent(
-    'LazyDrafts',
-    React.lazy(() => import('components/drafts')),
+const Drafts = makeAsyncComponent('Drafts', lazy(() => import('components/drafts')),
     (
         <div className='app__content'>
             <LoadingScreen/>
         </div>
     ),
 );
+const PermalinkView = makeAsyncComponent('PermalinkView', lazy(() => import('components/permalink_view')));
+const PlaybookRunner = makeAsyncComponent('PlaybookRunner', lazy(() => import('components/channel_layout/playbook_runner')));
+const ChannelIdentifierRouter = makeAsyncComponent('ChannelIdentifierRouter', lazy(() => import('components/channel_layout/channel_identifier_router')));
```

Comment:
Are you sure we want this component to be lazy? The channel view is the core of the app, and this is needed for it, so I would think we wouldn't want to split that one into a separate bundle

Reply:
Yes you are correct reverted